docs/setup/k8s/running-in-production-k8s.md

```diff
@@ -62,50 +62,64 @@ Below is a quick example of how to setup the a specific stable version of nuclio
+- Set `offline` to `true` to put Nuclio in "offline" mode.
+- Set `dashboard.baseImagePullPolicy` to `Never`.
+- Set `registry.pushPullUrl` to a registry URL that's reachable from your system.
+- <a id="air-gapped-envir-processor-n-onbuild-images"></a>Ensure that the processor and "onbuild" images are accessible to the dashboard in your environment, as they're required for the build process (either by `docker build` or [Kaniko](#using-kaniko-as-an-image-builder)).
+  You can achieve this using either of the following methods:
 
-- Most definitely [version freeze](#version-freezing). Also, set `*.image.pullPolicy` to `Never` or `IfNotPresent` to make sure k8s won't try to access the web to fetch images at any point in time.
-- Set `Values.offline=true` in the helm values, to put nuclio in "offline" mode. Set `dashboard.baseImagePullPolicy=Never`.
-- Needless to say, in this scenario, you will have to configure nuclio with `registry.pushPullUrl` which is reachable from your system.
-- The processor and onbuild images will also have to be accessible to the dashboard in your environment, as they are required for the building process - (by `docker build`, or [kaniko](#using-kaniko-as-an-image-builder)).
-  This can be tricky as you have to either make those images available to the k8s docker daemon or pull-able from a reachable registry, where they should be preloaded. Use `Values.registy.defaultBaseRegistryURL` to point nuclio at searching in your registry for those images, rather then at the default location of `quay.io/nuclio`.
-  To save some work on setting up a registry and preloading the onbuild images to it (or as a reference to what it should include) - take a look at the [prebaked-registry](https://github.com/nuclio/prebaked-registry).
-- For the Nuclio templates library to be available to you, you'll have to package that yourself, and have it served locally, somewhere within reach of your system. To point Nuclio to it, set `Values.dashboard.templatesArchiveAddress` to where you serve the templates.
+  - Make the images available to the Kubernetes Docker daemon.
+  - Preload the images to a registry that's accessible to your system, to allow pulling the images from the registry.
+    When using this method, set `registy.defaultBaseRegistryURL` to the URL of an accessible local registry that contains the preloaded images (thus overriding the default location of `quay.io/nuclio`, which isn't accessible in air-gapped environments).
+    <br/><br/>
+    > **Note:** To save yourself some work, you can use the [prebaked Nuclio registry](https://github.com/nuclio/prebaked-registry), either as-is or as a reference for creating your own local registry with preloaded images.
 
+- To use a Nuclio templates library (optional), package the templates into an archive, and set `dashboard.templatesArchiveAddress` to a local address for serving the archive, which is accessible to your system.
```

Comment:
```suggestion
- To use the Nuclio templates library (optional), package the templates into an archive and serve them via a local server of your doing on some address accessible to the system. Set `dashboard.templatesArchiveAddress` to the address of that local server.
```

Reply:
I changed it to the following — see commit f00f5c2:
> To use the Nuclio templates library (optional), package the templates into an archive; serve the templates archive via a local server whose address is accessible to your system; and set `dashboard.templatesArchiveAddress` to the address of this local server